### ISSUE TITLE
[Bug] Method may fail to close stream

### ISSUE THREAD

**Issue by kirinzhu** [apache/incubator-eventmesh#987]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/eventmesh/issues?q=is%3Aissue) and found no similar issues.

### Environment

Mac

### EventMesh version

master

### What happened

The method creates an IO stream object, does not assign it to any fields, pass it to other methods that might close it, or return it, and does not appear to close the stream on all paths out of the method.  This may result in a file descriptor leak.  It is generally a good idea to use a finally block to ensure that streams are closed.

### How to reproduce

In class org.apache.eventmesh.common.config.ConfigurationWrapper
In method org.apache.eventmesh.common.config.ConfigurationWrapper.load()
Need to close java.io.Reader
At ConfigurationWrapper.java:[line 85]

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

**Comment by ruanwenjun** [comment]
Good catch, it's better to use try-with-resource to close the resource.
